ted_data_eu/services/currency_convertor.py

```diff
@@ -0,0 +1,18 @@
+from datetime import datetime, date
+
```

Comment:
One file for a single simple service is not a good approach, I recommand to rename currency_convertor.py to data_normalisation.py for example.